presto/presto.sh

```diff
@@ -55,8 +50,8 @@ function wait_for_presto_cluster_ready() {
 
 function get_presto(){
   # Download and unpack Presto server
-  wget https://repo1.maven.org/maven2/com/facebook/presto/presto-server/${PRESTO_VERSION}/presto-server-${PRESTO_VERSION}.tar.gz
-  tar -zxvf presto-server-${PRESTO_VERSION}.tar.gz
+  wget https://storage.googleapis.com/starburstdata/presto/${PRESTO_MAJOR_VERSION}e/${STARBURST_PRESTO_VERSION}/presto-server-${STARBURST_PRESTO_VERSION}.tar.gz
+  tar -zxvf presto-server-${STARBURST_PRESTO_VERSION}.tar.gz
```

Comment:
Consider creating a symlink
```
ln -s "presto-server-${STARBURST_PRESTO_VERSION}" "presto-server"
```

so that you don't need the `${STARBURST_PRESTO_VERSION}` variable everywhere else

Reply:
Also, per bash code style, every variable should be in quotes

```suggestion
  tar -zxvf "presto-server-${STARBURST_PRESTO_VERSION}.tar.gz"
```

Reply:
Who's going to publish releases to https://storage.googleapis.com/starburstdata/ ? Can we use s3 url here instead?

Reply:
Me for now. I do not think we should use S3.
We should consider having our automation push to all three clouds.